projects/packages/sync/src/class-defaults.php

```diff
@@ -1374,5 +1374,5 @@ public static function is_multi_network() {
 	 *
 	 * @var int Bool-ish. Default 0.
```

Comment:
Minor thing - should this also be updated to 1?

```suggestion
	 * @var int Bool-ish. Default 1.
```

Reply:
Nice catch, Karen! Fixed via https://github.com/Automattic/jetpack/pull/45672/commits/ba4616c2495b0ce0a9c27f6bed0138b759920290